deluge/core/rpcserver.py

```diff
@@ -546,7 +546,9 @@ def emit_event(self, event):
         """
         log.debug('intevents: %s', self.factory.interested_events)
         # Find sessions interested in this event
-        for session_id, interest in self.factory.interested_events.items():
+        # The interested_events dict can change while iterating,
+        # therefore we need to iterate over a copy
+        for session_id, interest in self.factory.interested_events.copy().items():
```

Comment:
Well, the commit message is to explain why the change is needed. The comment is to ensure nobody removes the `copy` part later on because "It doesn't seem necessary".

Reply:
That's a reasonable point, okay let's remove original comment and make the new comment a short one-liner:

```diff
-        # Find sessions interested in this event
        for session_id, interest in self.factory.interested_events.items():
-        # The interested_events dict can change while iterating,
-        # therefore we need to iterate over a copy
+        # Use copy of `interested_events` since it can mutate while iterating.
```